IslandLanding/IslandLanding/Views/LosePage.xaml

```diff
@@ -0,0 +1,138 @@
+      <Image Source="background" Grid.RowSpan="4" Opacity="0.3"
+                  Aspect="Fill"/>
+      <Label 
+             Grid.Row="0" 
+             IsVisible="{Binding IsLosing}"
+             HorizontalOptions="Center"
+             VerticalOptions="Center"
+             TextColor="White"
+             FontSize="30"
+             VerticalTextAlignment="Center"
+             HorizontalTextAlignment="Center"
+             Margin="40,80,40,0" 
+             WidthRequest="313"
+             FontFamily="PatrickFont">
+        <Label.FormattedText>
+          <FormattedString>
+            <Span Text="Sorry , "/>
+            <Span Text="{Binding UserTag}"/>
+            <Span Text="You have landed on the lake."/>
+          </FormattedString>
+        </Label.FormattedText>
+      </Label>
+      <pancake:PancakeView Grid.Row="1"
+                         Margin="0,50,0,0"
+                           IsVisible="{Binding IsLosing}" Grid.RowSpan="2"
+                           HorizontalOptions="Center"
+                           VerticalOptions="Center"                   
+       >
+        <forms:AnimationView
+          Animation="Resources.parachut.json"
+          AnimationSource="EmbeddedResource"
+          AutoPlay="True"
+          HeightRequest="257"
+          WidthRequest="257" 
+          RepeatMode="Infinite"
+          />
+      </pancake:PancakeView>
+      <!--the other view for lose page -->
+      <forms:AnimationView 
+          Animation="Resources.lake.json" 
+          AnimationSource="EmbeddedResource" 
+          AutoPlay="True"  
+            Grid.Row="3"       
+          IsVisible="{Binding IsLosing,Converter={StaticResource InverseBool}}"
+          VerticalOptions="FillAndExpand"
+          HorizontalOptions="FillAndExpand"
+          RepeatMode="Infinite"
+         />
+
+      <Label   Margin="40,60,40,0"
+        Grid.Row="0" 
+           IsVisible="{Binding IsLosing,Converter={StaticResource InverseBool}}"
+        Text="You lost the game"
+             TextColor="White"
+             FontSize="40"
+             VerticalTextAlignment="Center"
+             HorizontalTextAlignment="Center"
+             FontFamily="PatrickFont"/>
+      <StackLayout  Grid.Row="1" 
+             IsVisible="{Binding IsLosing,Converter={StaticResource InverseBool}}"
+              Margin="{OnPlatform iOS='40,-100,40,0',Android='40,-160,40,0'}"
+             HorizontalOptions="Center"
+             VerticalOptions="Center">
+        
+        <Label 
+             TextColor="White"
+             FontSize="30"
+             Margin="0,3,0,0"
+             VerticalTextAlignment="Center"
+             HorizontalTextAlignment="Center"
+             FontFamily="PatrickFont">
+          <Label.FormattedText>
+            <FormattedString>
+              <Span Text="It took you "/>
+              <Span  Text="{Binding gameModel.MainTime}"/>
+              <Span Text=" seconds, "/>
+              <Span Text="which is "/>
```

Comment:
Is there a reason we dont merge these two spans?
`<Span Text=" seconds, which is "/>`